libjq.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: libjq
+URL: https://stedolan.github.io/jq/
+Description: Library to process JSON using a query language
+Version: @VERSION@
+Libs: -L${libdir} -ljq
```

Comment:
Does this need a run-path as well, for when `libjq` is installed somewhere other than a system location?

Reply:
We can fix that later, so I'll take it as is for now.